### PR TITLE
[CMake] Re-Enable include

### PR DESCRIPTION
**Pull request by Androbin**
Issue #3996 (and referenced tickets) are already fixed.

### CODE REVIEW

File: tensorflow/contrib/cmake/CMakeLists.txt [hunk 1]
```diff
@@ -387,8 +387,7 @@ endif()
 
 # Let's get to work!
 include(tf_core_framework.cmake)
-# NOTE: Disabled until issue #3996 is fixed.
-# include(tf_stream_executor.cmake)
+include(tf_stream_executor.cmake)
 if (tensorflow_ENABLE_GPU)
```
Comment (mrry) [inline tensorflow/contrib/cmake/CMakeLists.txt]: Can you remove this conditional block?

**Review by mrry — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/cmake/CMakeLists.txt (hunk 1 above)
Comment (Androbin) [reply tensorflow/contrib/cmake/CMakeLists.txt]: Oh, missed that one!

**Review by mrry — APPROVED** [review]
Thanks for the clean-up!

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by Androbin** [comment]
@sb2nov Is this the #14877 PIP package issue again?
`ImportError: No module named 'tensorflow.contrib.rnn.python.kernel_tests'`
`ImportError: No module named 'tensorflow.contrib.data.python.kernel_tests'`

**Comment by Androbin** [comment]
@sb2nov Shouldn't at least one of them be fixed as of 06a5cc4acd377815a0a6bbaac6ea0ab972f1c8fb?

**Comment by Androbin** [comment]
@caisq Test failures are well-known and should be unrelated to this PR.

**Pull request merged**